clang/test/CodeGenSYCL/unique-stable-name.cpp

```diff
@@ -8,7 +8,7 @@
 // CHECK: @[[LAMBDA_IN_DEP_INT:[^\w]+]] = private unnamed_addr constant [[DEP_INT_SIZE:\[[0-9]+ x i8\]]] c"_ZTSZ28lambda_in_dependent_functionIiEvvEUlvE23->12\00",
 // CHECK: @[[LAMBDA_IN_DEP_X:[^\w]+]] = private unnamed_addr constant [[DEP_LAMBDA_SIZE:\[[0-9]+ x i8\]]] c"_ZTSZ28lambda_in_dependent_functionIZZ4mainENKUlvE42->5clEvEUlvE46->16EvvEUlvE23->12\00",
 
-extern "C" void printf(const char*);
+extern "C" void printf(const char*) {}
```

Comment:
Why not 
```suggestion
SYCL_EXTERNAL extern "C" void printf(const char*);
```
?

Reply:
SYCL spec does not allow user to explicitly put SYCL_EXTERNAL attribute for function with raw pointer parameter/return type.